Mathlib/MeasureTheory/Function/LpSeminorm/Basic.lean

```diff
@@ -1155,6 +1167,25 @@ theorem _root_.Continuous.memℒp_top_of_hasCompactSupport
   apply memℒp_top_of_bound ?_ C (Filter.eventually_of_forall hC)
   exact (hf.stronglyMeasurable_of_hasCompactSupport h'f).aestronglyMeasurable
 
+section UnifTight
+
+/-- A single function that is `Memℒp f p μ` is tight with respect to `μ`. -/
+theorem Memℒp.exists_snorm_indicator_compl_lt {β : Type*} [NormedAddCommGroup β] (hp_top : p ≠ ∞)
+    {f : α → β} (hf : Memℒp f p μ) {ε : ℝ≥0} (hε : ε ≠ 0) :
```

Comment:
Why did you change `ENNReal` to `NNReal` here? It was more general before (even if trivially)

Reply:
¯\_(ツ)_/¯ I guess with that philosophy, there was no reason to change `UnifTight` to use `ENNReal` in the first place. I changed it here because it made some of the proofs I was reworking easier. Right now this doesn't contradict any convention for what type `ε` and `δ` in `Analysis` or `MeasureTheory`, since I don't see any unified convention.

Reply:
> I guess with that philosophy, there was no reason to change `UnifTight` to use `ENNReal` in the first place

I mean... yes, I agree? But that doesn't go against my comment above: Have easy to prove definitions, hard to prove theorems.

Reply:
OK, I can understand that. Apologies for being dense, but as I'm still learning about different Mathlib philosophies and conventions it'd be nice to hear some general guidelines or explanations along with patch change suggestions. Understanding some general guidelines better I think would also help to smoothly update the parent PR.

Reply:
That's why I keep on commenting here 😃

There's a lot of unwritten community knowledge that you will learn by just asking. We all wish it were written down, but nobody has done that yet. So please keep on asking questions.